sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/implementation/models/CommunicationError.java

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.communication.administration.implementation.models;
```

Comment:
This should be generated inside the new `identity` package.

Reply:
removed

Reply:
Actually my mistake on this one, it is generated by phone number SDK as well

Reply:
I try generating the phonenubmer in admin swagger. I don't see this class is re-generated. 
And after i remove it, the admin code still compile.

 autorest README.md --java --v4 --use=C:\Users\sacheu\source\GitHub\autorest.java --tag=phonenumber